commerce-notification-service/src/main/java/com/liferay/commerce/notification/model/impl/CommerceNotificationQueueEntryImpl.java

```diff
@@ -23,4 +30,23 @@
 	public CommerceNotificationQueueEntryImpl() {
 	}
 
+	@Override
+	public String getCommerceNotificationTemplateType() throws PortalException {
```

Comment:
@Commando18  Avoid putting methods in the ModelImpl that calls Util, let's avoid Singleton pattern usage